app/components/Sidebar/components/DocumentLink.js

```diff
@@ -96,7 +97,12 @@ class DocumentLink extends React.Component<Props> {
               state: { title: node.title },
             }}
             expanded={showChildren ? true : undefined}
-            label={node.title || "Untitled"}
+            label={
+              <Title bold={document?.modifiedSinceViewed || document?.new}>
+                {node.title || "Untitled"}
+                {document?.new && <Badge>New</Badge>}
```

Comment:
Could we add a Updated badge?

Reply:
I think this'll be too noisy, I think you'll quickly come to recognize bold as updated as it's a pretty standard treatment eg in email clients.

Reply:
When is updated i just set title bolder, as requested by @tommoor 
But we. could add like a dot, as in Slack

Reply:
Do not to forget to test with large file names... And create this effect when update and created.